docs/core/docker/building-net-docker-images.md

```diff
@@ -2,224 +2,255 @@
-- **Production:** How fast you can deploy and start your image. This image is small so it can quickly travel across the network from your Docker Registry to your Docker hosts. The contents are ready to run enabling the fastest time from Docker run to processing results. In the immutable Docker model, there's no need for dynamic compilation of code. The content you place in this image would be limited to the binaries and content needed to run the application. For example, the published output using `dotnet publish` which contains the compiled binaries, images, .js and .css files. Over time, you'll see images that contain pre-jitted packages.  
+When developing, building, and running containerized applications, we have different priorities.
+
+* **Development:**  The priority focuses on quickly iterate changes, and the ability to debug the changes. The size of the image isn't as important, rather can you make changes to your code and see them quickly?
+
+* **Build:** This image contains everything needed to compile your app, which includes the compiler and any other dependencies to optimize the binaries.  You use the build image to create the assets you place into a production image. The build image would be used for continuous integration, or in a build environment. This approach allows a build agent to compile and build the application (with all the required dependencies) in a build image instance. Your build agent only needs to know how to run this Docker image.
+
+* **Production:** How fast you can deploy and start your image? This image is small so network performance from your Docker Registry to your Docker hosts is optimized. The contents are ready to run enabling the fastest time from Docker run to processing results. Dynamic code compilation isn't needed in the Docker model. The content you place in this image would be limited to the binaries and content needed to run the application.
+
+    For example, the `dotnet publish` output contains:
+
+    * the compiled binaries
+    * .js and .css files
 
-Though there are multiple versions of the .NET Core image, they all share one or more layers. The amount of disk space needed to store or the delta to pull from your registry is much smaller than the whole because all of the images share the same base layer and potentially others.  
+
+The reason to include the `dotnet publish` command output in your production image is to keep its' size to a minimum.
+
+Some .NET Core images share layers between different tags so downloading the latest tag is a relatively lightweight process. If you already have an older version on your machine, this architecture decreases the needed disk space.
+
+When multiple applications use common images on the same machine, memory is shared between the common images. The images must be the same to be shared.
 
 ## Docker image variations
```

Comment:
I think it is important to mention what platforms the images support.  Right now these images work on both AMD64 Windows (RS1 and RS3) and AMD64 Linux.  Linux AMD32V7 support is in preview.

Reply:
Unhderstood. Would linking to the release notes work? That way the customer will have the latest info.

Reply:
Yes I think that would be great.